src/main/app/src/stores/rest-store.js

```diff
@@ -1,6 +1,6 @@
 import { computed, observable, action /*configure*/ } from 'mobx';
 import superagent from 'superagent';
-import { Common as C, Localizer as l } from '../tools/Utils';
+import { Localizer as l } from '../tools/Utils';
```

Comment:
```suggestion
import { Localizer as l } from '#/src/tools/Utils';
```

Reply:
Jees, näitä on hyvä muuttaa aina kun löytää :+1:

Reply:
Korjasin amendina